bciers/apps/registration/app/components/transfers/TransfersDataGridPage.tsx

```diff
@@ -0,0 +1,30 @@
+import TransferDataGrid from "./TransfersDataGrid";
+import { Suspense } from "react";
+import Loading from "@bciers/components/loading/SkeletonGrid";
+import { TransferRow, TransfersSearchParams } from "./types";
+import fetchTransferEventsPageData from "./fetchTransferEventsPageData";
+
+// 🧩 Main component
+export default async function TransfersDataGridPage({
+  searchParams,
+}: {
+  searchParams: TransfersSearchParams;
+}) {
+  // Fetch transfers data
+  const transfers: {
+    rows: TransferRow[];
+    row_count: number;
+  } = await fetchTransferEventsPageData(searchParams);
+  if (!transfers) {
+    return <div>No transfers data in database.</div>;
+  }
```

Comment:
Should we throw an error if `!transfers || "error" in transfers`? This condition indicates that something went wrong on the backend. If that's the case, we should display an error. Otherwise, we can show an empty data grid, similar to when there is simply no data to display. This approach helps us distinguish between an actual error and the absence of data.

Reply:
This is setup the same way as the other grids, I'll fix them too

Reply:
We started throwing errors in the registration forms at some point, but those grids are from before, I think. So don’t worry if it’s too much work; we can address it as technical debt.